src/App.vue

```diff
@@ -82,10 +82,13 @@ export default {
   box-sizing: border-box;
 }
 
+html {
```

Comment:
Maybe we wanna add this: https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior#Mind_users_with_motion_sickness